src/evmTest/scala/io/iohk/ethereum/vm/MinimumViableTokenSpec.scala

```diff
@@ -31,7 +31,7 @@ class MinimumViableTokenSpec extends FreeSpec with Matchers {
       val (_, contract) = deployContract("MinimumViableToken", creatorAddress = sender, constructorArgs = Seq(100))
 
       val transferRes = contract.transfer(receiver, 200).call(sender = sender)
-      transferRes.error shouldBe Some(InvalidOpCode) // TODO: change to invalid jump destination
+      transferRes.error shouldBe Some(InvalidOpCode(0xfd.toByte))
```

Comment:
well that's what solidity generates, and seems we don't have a constant for 0xfd (revert op?)